Resources/Prototypes/GameRules/roundstart.yml

```
@@ -331,6 +333,23 @@
     - groups: TraitorObjectiveGroups
     maxDifficulty: 5
 
+# Goobstation - order swapped since admin verbs use .Last() for some reason
+# TODO: Fix that 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)